strategy/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "event-listener-strategy"
+version = "0.1.0"
+edition = "2018"
```

Comment:
Can we just go with 2021 maybe? or do we want it consistent across all smol-rs crates?

Reply:
I want to use this in `async-lock` and `async-channel`, so it needs to be 2018 edition.

Reply:
Edition is strictly specific to the crate itself, doesn't affect the dependent crates.

Reply:
what's the implied MSRV of edition 2021?

Reply:
[1.56](https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html) apparently.

Reply:
hmm.. that would be problematic cause MSRV of event-listener itself is 1.39. So nm my original suggestion here but we should declare our MSRV in this new crate to ensure it's not implicitly incompatible (or accidently becomes so in the future).